xwiki-platform-core/xwiki-platform-web/xwiki-platform-web-war/src/main/webapp/resources/js/xwiki/widgets/modalPopup.css

```diff
@@ -42,34 +42,27 @@
     width: 400px; /* Should be: @modal-md */
   }
 }
-.xdialog-box .xdialog-content {
+.xdialog-box .xdialog-content,
+.xdialog-box > .xdialog-close {
   margin: .8em;
 }
-.xdialog-close {
+.xdialog-box > .xdialog-close {
   float: right;
-  margin-right: .8em;
 }
 .xdialog-title .xdialog-close {
   font-size: 1.25em;
-  margin-right: -0.56em;
-  margin-top: -0.24em;
   color: $theme.panelHeaderTextColor;
 }
 .xdialog-title {
-  font-size: 80%;
-  font-weight: bold;
-  padding: .3em 1.2em .3em .8em;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 1rem;
   background-color: $theme.panelHeaderBackgroundColor;
-  #css3_backgroundLinearGradient({
-     'to': 'bottom',
-     'colors': [
-      {'color': $theme.panelHeaderGradientColor, 'position': '0%'},
-      {'color': $theme.panelHeaderBackgroundColor, 'position': '100%'}
-    ]
-  })
   border-radius: 7px 7px 0 0;
+  font-weight: bold;
+  font-size: 110%;
```

Comment:
Another dialog style we have are the dialogs in the tour:

![image](https://github.com/xwiki/xwiki-platform/assets/198317/bd1e2306-763a-4d35-bc0c-d0a7a6e5e802)

Maybe that style is more suitable for small dialogs?

Reply:
Addressed some of those consistency issues in https://github.com/xwiki/xwiki-platform/pull/2703/commits/89ff72ab9e0fb59a2546c1baf242026bb94c9540 :+1: 
Note that there is still a few differences:
* Structure is different inside of the block, so there might be special cases where there is no good alternative (e.g. third screenshot thereafter, the header is just the button, everything else is just content - even what looks like a title...)
* We don't have access to LESS variables, so I picked the closest we had in the old theme variables, or hard coded (needed for the border-radius and font-size)

![21650-afterPR-23](https://github.com/xwiki/xwiki-platform/assets/28761965/5dcccbb9-6355-4d4f-9660-c5c5dda448ec)
![21650-afterPR-22](https://github.com/xwiki/xwiki-platform/assets/28761965/d09e09b3-391c-4fb4-8394-244c6bfbc25b)
![21650-afterPR-21](https://github.com/xwiki/xwiki-platform/assets/28761965/cb14c82f-5aa7-49de-90a5-fd03ce8cd4d3)

Reply:
> Another dialog style we have are the dialogs in the tour:
> 
> ![image](https://private-user-images.githubusercontent.com/198317/289104095-bd1e2306-763a-4d35-bc0c-d0a7a6e5e802.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDIwNTAwNDAsIm5iZiI6MTcwMjA0OTc0MCwicGF0aCI6Ii8xOTgzMTcvMjg5MTA0MDk1LWJkMWUyMzA2LTc2M2EtNGQzNS1iYzBjLWQwYTdhNmU1ZTgwMi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBSVdOSllBWDRDU1ZFSDUzQSUyRjIwMjMxMjA4JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDIzMTIwOFQxNTM1NDBaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT00OTZlOTBmZDBhZDYwZGMzNDYyMzQ3YzFmZmUyNmY2MWM1MGVkNWIxMjA4ZTdiODYzOTg0NmRlYmI0ODEyNWFiJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.7IC3Q7lr6W7KVHuZjL8vAVS1qHchIrrv41NVfZn2VaM)
> 
> Maybe that style is more suitable for small dialogs?

Sorry, I didn't get the update with your second message until too late.
Note that this kind of dialog might be used with some large content too. The third picture is such a dialogue, be it quite heavily malformed.
Let me know if the style above is too hefty, I'll revert the last commit and see if I can get something closer to the tour dialog.

If not, it might be interesting to open an improvement ticket to update the tour dialog style to match the other dialogs.